File: 0941-sort-array-by-parity/0941-sort-array-by-parity.py
class Solution:
    def sortArrayByParity(self, nums: List[int]) -> List[int]:
        oa=[]
        ea=[]
        for i in range(len(nums)):
            if nums[i]%2==0:
                ea.append(nums[i])
            else:
                oa.append(nums[i])
        return ea+oa

File: 0941-sort-array-by-parity/0941-sort-array-by-parity.java
class Solution {
    public int[] sortArrayByParity(int[] nums) {
        List<Integer> ea= new ArrayList<>();
        List<Integer> oa= new ArrayList<>();
        for (int i=0;i<nums.length;i++){
            if (nums[i]%2==0){
                ea.add(nums[i]);
            }
            else{
                oa.add(nums[i]);
            }
        }
        ea.addAll(oa);
        int [] result = new int[ea.size()];
        for (int i=0;i<ea.size();i++){
            result[i]=ea.get(i);
        }
        return result;
    }
}